docs/setup/mac.md

```diff
@@ -43,6 +43,15 @@ Start a new terminal to pick up your `.bash_profile` changes.
 
 **Note**: The leading slash `\` is required to prevent `$PATH` from expanding during the concatenation. Remove the leading slash if you want to run the export command directly in a terminal.
 
+**Note**: Since `zsh` became the default shell in macOS Catalina, run the following commands to add VS Code to your path:
+
+```zsh
+cat << EOF >> ~/.zshrc
+# Add Visual Studio Code (code)
+export PATH="/Applications/Visual Studio Code.app/Contents/Resources/app/bin:$PATH"
+EOF
+```
```

Comment:
Instead of duplicating everything the note could call out to replace `.bash_profile` with `.zshrc`, `.profile`, etc. Also is there a reason you don't use `.zprofile`?

Reply:
The reason not adding this to `.zprofile` is that it is sufficient to have it in an interactive shell instead of sourcing it in a login shell. I suggest changing `.bash_profile` to `.bashrc` and adding a note this could replace with `.zshrc` if that's the default shell.

Reply:
MacOS terminals by default run in a login shell though, not on actual login? I'm pretty sure I've had to investigate issues with scripts in `.bashrc` not being run in the terminal because it wasn't being sourced from `.bash_profile`, it seems safer to leave it as is with the profile.